gwpy/utils/tests/test_lal.py

```diff
@@ -52,11 +53,36 @@ def test_find_typed_function():
     try:
         import lalframe
     except ImportError:  # no lalframe
-        pass
-    else:
-        utils_lal.find_typed_function(
-            'REAL4', 'FrStreamRead', 'TimeSeries',
-            module=lalframe) is lalframe.FrStreamReadREAL4TimeSeries
+        return
+    utils_lal.find_typed_function(
+        'REAL4', 'FrStreamRead', 'TimeSeries',
+        module=lalframe) is lalframe.FrStreamReadREAL4TimeSeries
+
+
+@pytest.mark.parametrize(
+    'dtype',
+    [
+        numpy.int16, numpy.int32, numpy.int64,
+        numpy.uint16, numpy.uint32, numpy.uint64,
+        numpy.float32, numpy.float64,
+        numpy.complex64, numpy.complex128,
+    ]
+)
+def test_from_lal_type(dtype):
+    func = utils_lal.find_typed_function(dtype, 'Create', 'TimeSeries')
+    lalts = func(None, None, 0, 1., None, 1)
+    assert utils_lal.from_lal_type(lalts) is dtype
+    assert utils_lal.from_lal_type(type(lalts)) is dtype
+
+
+@pytest.mark.parametrize(
+    'name',
+    ['XXX', 'XXXCOMPLEX64ZZZ', 'INNT2', 'INT5', 'REAL42'],
+)
```

Comment:
Can you move the parametrize list to one per line, possibly like this:

```suggestion
@pytest.mark.parametrize('name', [
    'XXX',
    'XXXCOMPLEX64ZZZ',
    'INNT2',
    'INT5',
    'REAL42',
])
```